types/devicefarmer__stf-syrup/devicefarmer__stf-syrup-tests.ts

```diff
@@ -0,0 +1,38 @@
+import Syrup from "@devicefarmer/stf-syrup";
+import Promise from "bluebird";
```

Comment:
`bluebird` in 2025, wow!

Reply:
Yeah, I know. We are working on getting rid of it :)